### ISSUE TITLE
Improve the README to make it easy 

### ISSUE THREAD

**Issue by stickycode** [Branchout/branchout#34]
The README should be in 4 sections

What do you want to do with a Projected Monorepo

I want to use one
I want to create one
How does it work with maven, gradle, node

I want to contribute to Branchout

**Comment by fredcooke** [comment]
Yes, I was thinking about tackling this first as I was looking for basic "create a new one" docs and found bugger all. I was thinking along the same lines "create a new one" and "example workflow for maven" and "example workflow for adding a repo" etc.

**Comment by stickycode** [comment]
Good idea
